src/t2iapi/context/service.proto

```diff
@@ -105,6 +105,26 @@ service ContextService {
   t2iapi.context.CreateContextStateWithAssocAndSpecificValidatorRequest)
       returns (t2iapi.context.CreateContextStateWithAssociationResponse);
 
+  /*
+  Manipulate a given context state (specified by state_handle), such that
+  - has the given context_association value,
+  - has the given number of pm:Validators (num_validators)
+  - its pm:Identification changes so that it differs from the last 2 pm:Identifications
+    of this particular context state.
+  Shall return the handle of the context state. This context state shall either be
+  newly created or it shall be adapted from a previously existing context state of the same descriptor, by
+  - disassociating it (in case it was associated)
+  - changing its Validators and Identifications
+  - setting its @contextAssociation to the given value.
+  Note that changing the pm:Validator or pm:Identification elements of a context state might
```

Comment:
From my point of view this Note is not needed. The product has to behave correctly otherwise we report a failure.

Reply:
This is just a point that the implementor of the manipulation has to keep in mind.
I am not sure which Requirements would be violated if he did it differently and I don't know if we have tests for them.
And even if we have, we cannot be sure that these tests are enabled...

Reply:
Good Point. However, we should make sure that SDCcc notices when they do it wrong...

Reply:
As far as I see, the next Release of SDCcc will notice this.

Reply:
Removed the Note.